httomo/runner/dataset.py

```diff
@@ -178,6 +179,11 @@ def is_full(self) -> bool:
         """Check if the dataset is the full global data"""
         return False
 
+    @property
+    def is_full(self) -> bool:
+        """Check if the dataset is the full global data"""
+        return False
+
```

Comment:
Potentially a duplicated method, the diff shows that `is_full()` is already there above the newly added lines?

Reply:
indeed - that was a merge error. Fixed.